cmd/agent-sandbox-controller/main.go

```diff
@@ -61,8 +66,23 @@ func main() {
 		utilruntime.Must(extensionsv1alpha1.AddToScheme(scheme))
 	}
 
+	metricsOpts := metricsserver.Options{
+		BindAddress: metricsAddr,
+	}
+	if enablePprof {
+		setupLog.Info("pprof enabled")
+		metricsOpts.ExtraHandlers = map[string]http.Handler{
+			"/debug/pprof/":        http.HandlerFunc(pprof.Index),
+			"/debug/pprof/cmdline": http.HandlerFunc(pprof.Cmdline),
+			"/debug/pprof/profile": http.HandlerFunc(pprof.Profile),
+			"/debug/pprof/symbol":  http.HandlerFunc(pprof.Symbol),
```

Comment:
The `/debug/pprof/symbol` endpoint can be used to look up program counters, potentially giving an attacker more information about the binary's structure to aid in exploiting other vulnerabilities. This endpoint is not typically needed for standard profiling and should be disabled unless required for a specific debugging scenario.